client/src/components/ProfileListings/ProfileListings/ProfileListing.tsx

```diff
@@ -0,0 +1,202 @@
+  const handleSearchIconClick = () => updateSnackBarMessage(tooltipMessage);
+
+  return (
+    <Grid container direction="column" alignItems="center" className={classes.searchContainer}>
+      <Grid item container alignItems="center" direction="column">
+        <Typography align="center" variant="h3" component="h1" className={classes.bold}>
+          Your search Results
+        </Typography>
+        <form onSubmit={(e) => e.preventDefault()}>
+          <Grid container justify="center" className={classes.formContainer}>
+            <Autocomplete
+              id="asynchronous address search"
+              options={uniqueAddress}
+              onInputChange={(e, value) => setAddressText(value)}
+              noOptionsText="No Match Found"
+              freeSolo
+              autoSelect
+              blurOnSelect
+              loading={isLoading}
+              className={clsx(classes.input, classes.autocomplete)}
+              renderInput={(params) => (
+                <TextField
+                  {...params}
+                  placeholder="Location e.g Toronto"
+                  color="warning"
+                  InputProps={{
+                    ...params.InputProps,
+                    startAdornment: (
+                      <Tooltip title={tooltipMessage}>
+                        <SearchIcon color="primary" className={classes.iconTooltip} onClick={handleSearchIconClick} />
+                      </Tooltip>
+                    ),
+                  }}
+                />
+              )}
+            />
+            <Box className={classes.input}>
+              <DatePicker
+                value={dropInDate}
+                onChange={(value) => setDropInDate(value)}
+                renderInput={(params) => (
+                  <TextField
+                    {...params}
+                    color="warning"
+                    inputProps={{ ...params.inputProps, className: classes.datePicker }}
+                    label="Drop in"
+                  />
+                )}
+              />
+            </Box>
+            <Box className={classes.input}>
+              <DatePicker
+                value={dropOffDate}
+                onChange={(value) => setDropOffDate(value)}
+                renderInput={(params) => (
+                  <TextField
+                    {...params}
+                    color="warning"
+                    inputProps={{ ...params.inputProps, className: classes.datePicker }}
+                    label="Drop off"
+                  />
+                )}
+              />
+            </Box>
+          </Grid>
+        </form>
+      </Grid>
+      {isLoading ? (
+        <Box>
+          <Typography variant="h5" align="center">
+            Getting search results
+          </Typography>
+          {[
+            ['Address', addressText],
+            ['Drop in', getDateString(dropInDate)],
+            ['Drop off', getDateString(dropOffDate)],
+          ].map((message) => (
```

Comment:
I think you can simplify this by adding the text directly to the `Typography` component :)

`<Typography variant="body2"> Address: {addressText} <Typography>`